pkg/portmap/portmap.go

```diff
@@ -24,6 +25,16 @@ func ForwardPorts(t time.Duration) error {
 		// Detect ports for forward
 		newports, err := iptables.GetPorts()
 		if err != nil {
+			// iptables exiting with an exit status of 4 means there
+			// is a resource problem. For example, something else is
+			// running iptables. In that case, we can skip trying it for
+			// this loop. You can find the exit code in the iptables
+			// source at https://git.netfilter.org/iptables/tree/include/xtables.h
+			if strings.Contains(err.Error(), "exit status 4") {
```

Comment:
This didn't work for me, but if it did, then you would want a `time.Sleep(t)` before the `continue` to avoid spamming the logfile.

Reply:
Updated with the sleep. Good catch.